kratos/python_scripts/analysis_stage.py

```diff
@@ -330,6 +330,16 @@ def _GetOrderOfOutputProcessesInitialization(self):
         """
         return []
 
+    def _CheckDeprecatedOutputProcesses(self, list_of_processes):
+        deprecated_output_processes = []
+        for process in list_of_processes:
+            if issubclass(type(process), KratosMultiphysics.OutputProcess):

```

Comment:
Im not sure, is `issubclass` recursive? I mean, would this return true if a process is derived from another process which is finally derived from an output process?

Reply:
Is recursive. I've added a test to show that behavior